edx_proctoring/callbacks.py

```diff
@@ -152,12 +155,17 @@ def get(self, request, attempt_code):  # pylint: disable=unused-argument
             )
 
         update_exam_attempt(attempt['id'], last_poll_timestamp=timestamp, last_poll_ipaddr=ip_address)
+        time_remaining_seconds = get_time_remaining_for_attempt(attempt)
+        polling_interval = constants.DEFAULT_CLIENT_POLLING_INTERVAL
+        if time_remaining_seconds < constants.EXAM_CONCLUDING_INTERVAL:
```

Comment:
When time runs out, the status becomes `timed_out`, I believe. There's a way of generalizing all the statuses into 'complete' statuses. There's a `ProctoredExamStudentAttemptStatus.is_completed_status` that we can use.